src/Illuminate/Mail/Transport/SparkPostTransport.php

```diff
@@ -54,7 +54,7 @@ public function send(Swift_Mime_SimpleMessage $message, &$failedRecipients = nul
 
         $message->setBcc([]);
 
-        $response = $this->client->post('https://api.sparkpost.com/api/v1/transmissions', [
+        $response = $this->client->post(config('mail.endpoints.sparkpost', 'https://api.sparkpost.com/api/v1/transmissions'), [
```

Comment:
This should be `services.sparkpost.endpoint`.

Reply:
You can't just call the foundation config function from the mail component. It doesn't exist.

Reply:
What @GrahamCampbell means is: The helper `config()` is not available if you use the package `illuminate/mail` individually.